src/main/java/frc/robot/RobotConstants.java

```diff
@@ -133,12 +133,13 @@ public interface POV {
         }
     }
     public interface CAN {
-        public static final int FRONT_LEFT_MOTOR = 5;
-        public static final int REAR_RIGHT_MOTOR = 6;
-        public static final int FRONT_RIGHT_MOTOR = 7;
-        public static final int REAR_LEFT_MOTOR = 8;
-        public static final int ELBOW_MOTOR_ID = 3;
-        public static final int TELESCOPE_MOTOR_ID = 2;
+
+        public static final int FRONT_LEFT_MOTOR = 1;
+        public static final int REAR_RIGHT_MOTOR = 3;
+        public static final int FRONT_RIGHT_MOTOR = 2;
+        public static final int REAR_LEFT_MOTOR = 4;
+        public static final int ELBOW_MOTOR_ID = 0;
```

Comment:
These motor IDs have a bug we found when testing.  They need to be:

public static final int FRONT_LEFT_MOTOR = 1;
public static final int FRONT_RIGHT_MOTOR = 2;
public static final int REAR_LEFT_MOTOR = 3;
public static final int REAR_RIGHT_MOTOR = 4;

Reply:
@mandrews281 ok i updated to these